lib/next_ls/runtime/sidecar.ex

```diff
@@ -15,14 +15,23 @@ defmodule NextLS.Runtime.Sidecar do
   end
 
   def handle_info({:tracer, payload}, state) do
+    payload = replace_nil(payload, :start_line, 1)
```

Comment:
i think we just need it on the references, not symbols

Reply:
Yup (checked it locally).